io/eolearn/io/local_io.py

```diff
@@ -448,13 +444,13 @@ def execute(self, eopatch=None, *, filename=None):
                 with filesystem.openbin(path, "r") as file_handle:
                     with rasterio.open(file_handle) as src:
 
-                        data_bbox = BBox(src.bounds, CRS(src.crs.to_epsg()))
+                        boundless = True
                         if eopatch.bbox is None:
-                            eopatch.bbox = data_bbox
+                            boundless = False
```

Comment:
As mentioned in the discussion, I agree with the conclusion, but the fix here is empirical, since I was getting issues when using the `batch_to_eopatch` pipeline from `eo-grow`. Perhaps this points to still not completely understanding how the tool should be used.

Reply:
Ok, then let's have it this way.